cli/chex/dexpi.xml

```diff
@@ -0,0 +1,4924 @@
+<?xml version='1.0' encoding='UTF-8'?>
```

Comment:
Does this file need to be included here? I can't find any usages of it.

Reply:
I mixed the documentation for these two functions! Corrected it now

Reply:
I forgot to remove the file I used for testing the CLI. Will fix asap